$MirOS: ports/security/gnupg/patches/patch-scripts_autogen_sh,v 1.2 2006/10/02 02:05:45 tg Exp $
--- scripts/autogen.sh.orig	Wed Sep 13 07:19:30 2006
+++ scripts/autogen.sh	Sun Mar 11 20:13:05 2007
@@ -243,9 +243,6 @@ fi
 if check_version $AUTOMAKE $automake_vers_num $automake_vers; then
   check_version $ACLOCAL $automake_vers_num $automake_vers automake
 fi
-if check_version $GETTEXT $gettext_vers_num $gettext_vers; then
-  check_version $MSGMERGE $gettext_vers_num $gettext_vers gettext
-fi
 
 if test "$DIE" = "yes"; then
     cat <<EOF
